docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -73,17 +74,72 @@ the light client isn’t expired.
 
 After having successfully verified a misbehaviour, the endpoint will execute the jailing and slashing of the malicious validators similarly as in the evidence module. 
 
-### Current limitations:
 
-- This only handles light client attacks, not double signing. In the future, we will add the code to also verify double signing.
+### Double Signing Attack
+
+A double signing attack, also known as an equivocation,
+occurs when a validator sends two different votes for a block in the same round of a consensus instance. 
+The CometBFT consensus operates with multiple rounds of voting at each block height 
+to determine the next block, and voting twice in the same round is strictly prohibited.
+
+When a node observes two votes from the same peer, it will use these two votes to create 
+a [`DuplicateVoteEvidence`](https://github.com/cometbft/cometbft/blob/2af25aea6cfe6ac4ddac40ceddfb8c8eee17d0e6/types/evidence.go#L35)
```

Comment:
Maybe we can refer to a specific release of CometBFT, interchain-security, and Cosmos SDK when using links.